README.md

```diff
@@ -88,9 +88,9 @@ Tests for Z3-Noodler are located in [src/test/noodler](src/test/noodler).
 
 ## Licensing
 
-The code of Z3-Noodler is distributed under the MIT License. See [LICENSE.md](./LICENSE.md).
+Z3-Noodler is distributed under the MIT License. See [LICENSE.md](./LICENSE.md).
 
-Z3-Noodler is a derivative work of Z3, distributed under the MIT License. See [LICENSE_Z3.txt](./LICENSE_Z3.txt).
+Z3-Noodler is a derivative work of Z3 SMT solver, distributed under the MIT License. See [LICENSE_Z3.txt](./LICENSE_Z3.txt).
```

Comment:
Z3-Noodler is a derivative of the SMT solver Z3. Z3-Noodler is distributed under the MIT License. 

(don't like "Z3 SMT solver", ugly english, and it was not clear what the second sentence was refering to, Z3 ro Z3-Noodler)

Reply:
` Z3 SMT solver` is the official name from Z3 materials.

Reply:
And this is supposed to mention the Z3 original licence, not the Z3-Noodler license.

Reply:
Fixed.